test/stdlib/Random.swift

```diff
@@ -0,0 +1,225 @@
+// RUN: %target-run-simple-swift
+// REQUIRES: executable_test
+
+import StdlibUnittest
+
+let RandomTests = TestSuite("Random")
+
+// Basic random numbers
+
+RandomTests.test("basic random numbers") {
+  let randomNumber1 = Int.random(in: .min ... .max)
+  let randomNumber2 = Int.random(in: .min ... .max)
+  expectTrue(randomNumber1 != randomNumber2)
+
+  let randomDouble1 = Double.random(in: 0 ..< 1)
+  expectTrue(randomDouble1 < 1 && randomDouble1 >= 0)
+  let randomDouble2 = Double.random(in: 0 ..< 1)
+  expectTrue(randomDouble1 < 1 && randomDouble2 >= 0)
```

Comment:
Line 18 should be `expectTrue(randomDouble2 < 1 && randomDouble2 >= 0)`

Maybe the "basic random numbers" test should use two arrays of numbers?